src/components/Streamer.tsx

```
@@ -10,13 +10,29 @@ import ContentViewer from "./ContentViewer"
 import { Document, Packer, Paragraph, TextRun } from "docx"
 import { Tabs, TabsList, TabsTrigger } from "@/components/ui/tabs"
 import { commit, undo, redo, history } from "../utils/immerHistory"
+import { supabase } from "@/common/supabase";
+
+function saveToSupabase(analysisId: string,
+    pagesContent: any[],
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)